Packs/CommonTypes/IncidentFields/incidentfield-Account_Member_Of.json

```diff
@@ -0,0 +1,30 @@
+{
+    "associatedToAll": true,
+    "caseInsensitive": true,
+    "cliName": "accountmemberof",
+    "closeForm": false,
+    "content": true,
+    "editForm": true,
+    "group": 0,
+    "hidden": false,
+    "id": "incident_accountmemberof",
```

Comment:
What is that field?

Reply:
Member Of = Account's group
Tried to align that with AD as much as possible but I can change that ( it might require to change in the playbooks as well).